src/login.c

```diff
@@ -573,9 +573,9 @@ int main (int argc, char **argv)
 #ifdef RLOGIN
 	if (rflg) {
 		assert (NULL == username);
-		username = XMALLOC(USER_NAME_MAX_LENGTH + 1, char);
-		username[USER_NAME_MAX_LENGTH] = '\0';
-		if (do_rlogin (hostname, username, USER_NAME_MAX_LENGTH, term, sizeof term)) {
+		username = XMALLOC(sysconf(_SC_LOGIN_NAME_MAX) + 1, char);
```

Comment:
nitpick, but I'd prefer you use a temp size_t to store sysconf(_SC_LOGIN_NAME_MAX) and reuse in the below three lines.  (Same in the next block)

Reply:
Done